src/platforms/hosted/jlink.c

```diff
@@ -317,15 +327,34 @@ static bool jlink_get_version(void)
 	return true;
 }
 
+static bool jlink_get_extended_capabilities(void)
+{
+	uint8_t buffer[32U];
+
```

Comment:
Please remove this excess new line to better match the style of similar functions in the file